frontend/packages/component-library/src/actionBlock/ActionBlock.tsx

```diff
@@ -61,6 +62,12 @@ export const getButtons = (
   );
 };
 
+export const getTag = (tag: string) => {
+  return (
+    <BodyFourText className={classNames(moduleStyles.tag)}>{tag}</BodyFourText>
```

Comment:
I'm not using the `Tag` component here because it uses hooks which would require us to render the action blocks client side in Contentful. Since it's just decorative it's fine to use a `p` tag.

Reply:
This works, but this also worries me long term. Can we please create a ticket for 'Use DSCO Tag component in ActionBlock components intstead of current custom implementation (once DSCO Tag component will be supporting SSR)'? (not sure if it should live in cms or dsco board though)
Realistically that may happen after either we update bundling/styles structure of existing DSCO or use MUI for Tag component (whichever comes first)

cc @bethanyaconnor @moshebaricdo

Reply:
This doesn't need the features the Tag component offers so I think it's overkill to use it here. But I'm also like a 6/10 level opinion on this and can be persuaded to use the MUI one if it doesn't require client side rendering.